here-naksha-app-service/src/test/java/com/here/naksha/app/service/DeleteFeatureTestHelper.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2017-2023 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+package com.here.naksha.app.service;
+
+import static com.here.naksha.app.common.TestUtil.*;
+import static com.here.naksha.app.common.TestUtil.HDR_STREAM_ID;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.here.naksha.app.common.NakshaTestWebClient;
+import com.here.naksha.lib.core.models.naksha.Space;
+import java.net.http.HttpResponse;
+import java.util.UUID;
+import org.jetbrains.annotations.NotNull;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+
+public class DeleteFeatureTestHelper {
+
+  final @NotNull NakshaApp app;
+  final @NotNull NakshaTestWebClient nakshaClient;
+
+  public DeleteFeatureTestHelper(final @NotNull NakshaApp app, final @NotNull NakshaTestWebClient nakshaClient) {
+    this.app = app;
+    this.nakshaClient = nakshaClient;
+  }
+
+  void tc0900_testDeleteFeatures() throws Exception {
+    // Test API : DELETE /hub/spaces/{spaceId}/features
+    final String streamId = UUID.randomUUID().toString();
+
+    // Preparation: create storage, event handler, space, and initial features
+    final String storage = loadFileOrFail("TC0900_deleteFeatures/create_storage.json");
+    nakshaClient.post("hub/storages", storage, streamId);
+    final String handler = loadFileOrFail("TC0900_deleteFeatures/create_handler.json");
```

Comment:
I suggest adding basic response code assertion at all steps, so if storage creation fails for example, then we will know before create_handler step.
`assertEquals(200, response.statusCode(), "ResCode mismatch. Failed creating Storage");`